articles/basics-of-convolution-neural-networks/index.md

```diff
@@ -0,0 +1,168 @@
+### Prerequisites
+
+This article assumes a basic understanding of Machine Learning (ML) and Deep Learning (DL). For an introduction to ML and DL, feel free to check out my previous [article](/https://www.section.io/engineering-education/differences-between-artificial-intelligence-machine-learning-and-deep-learning/)
+
+### Introduction
+
+Vision is one of the key senses in humans. We rely on vision a lot, from recognizing physical objects to navigating in the real world. It's a huge part of our lives.
+The way humans see images isn't the same way computers see them. Computers see images as an array of numbers. As numbers in computers are stored as 8-bit integers, these allow a range of pixel values from 0 to 255 to be displayed. These values correspond to the pixel brightness in the image.
+
+Let's take an example of an image with the number 8 shown below:
+
+![Image as an array](/engineering-education/basics-of-convolution-neural-networks/image-as-array.PNG)<br>
+
+*[Image Source: The MNIST Database](http://yann.lecun.com/exdb/mnist/)*
+
+In computer science, a pixel value of 0 represents black (dull) while 255, white (bright). As seen in the image, the pixels surrounding the image tend to be much brighter (closer to 255) than pixels away from it. This is how a computer sees an image. 
+
+Convolution Neural Networks (CNNs) builds on this idea. 
+
+### Terminologies
+
+1. Parameter
+
+These are placeholders that change during training. Weights and Biases are the two main parameters in CNNs.
+
+2. Hyperparameter
+
+A hyperparameter is also a placeholder, but unlike the parameter, hyperparameters need to set before training starts. The learning rate is one of the important hyperparameters that need to be set before training starts. 
+
+3. Kernel
+
+A kernel is an integral component of the structure of a CNN. It consists of a set of learnable parameters that are applied to the operations of convolutions. 
+
+4. Stride
+
+Stride refers to the number of pixels a convolution filter moves. It can be compared to a sliding window. The image below shows the stride, S moving with a pixel of 2
+
+![Stride, S](/engineering-education/basics-of-convolution-neural-networks/stride.PNG)<br>
+
+*[Image Source: Stanford](https://stanford.edu/~shervine/teaching/cs-230/cheatsheet-convolutional-neural-networks)*
+
+### What is Convolution Neural Networks (CNN)?
+
+Convolution Neural Networks (CNN) is a class of deep learning techniques popularly use to solve computer vision tasks.
+CNN learns the spatial features of inputs through a process known as backpropagation, as we will see in the image below. Backpropagation can be described as the backward propagation of errors. It is the core algorithm behind how neural networks learn. Learn more about it in this [video](/https://www.youtube.com/watch?v=Ilg3gGewQ5U/).
```

Comment:
Factual error: CNN learns the spatial features through convolution operation. Backpropagation is used to update the weights and minimize/maximize the objective function.